client/src/www/model/platform_error.ts

```diff
@@ -158,7 +200,7 @@ export class PlatformError extends CustomError {
    *   throw PlatformError.parseFrom(e);
    * }
    */
-  static parseFrom(errObj: string | Error | unknown): PlatformError {
+  static parseFrom(errObj: string | Error | unknown): Error {
```

Comment:
Please update all `PlatformError` in the above comments to `Error`.

Reply:
Done